debezium-core/src/main/java/io/debezium/util/Strings.java

```diff
@@ -49,7 +49,7 @@
      * @param factory the factory for creating string items into filter matches; may not be null
      * @return the list of objects included in the list; never null
```

Comment:
Maybe the docs should be changed to reflect they do not return list but sets - just a cosmektic change to keep things consistent.

Reply:
done. Needed changes at a couple more places. Have made those as well.